dist/setup/index.js

```diff
@@ -131520,9 +131520,9 @@ class SapMachineDistribution extends base_installer_1.JavaBase {
         return __awaiter(this, void 0, void 0, function* () {
             const platform = this.getPlatformOption();
             const arch = this.distributionArchitecture();
-            let fetchedReleasesJson = yield this.fetchReleasesFromUrl('https://sap.github.io/SapMachine/assets/data/sapmachine-releases-all.json');
+            let fetchedReleasesJson = yield this.fetchReleasesFromUrl('https://sapmachine.io/assets/data/sapmachine-releases-all.json');
             if (!fetchedReleasesJson) {
-                fetchedReleasesJson = yield this.fetchReleasesFromUrl('https://api.github.com/repos/SAP/SapMachine/contents/assets/data/sapmachine-releases-all.json?ref=gh-pages', (0, util_1.getGitHubHttpHeaders)());
+                fetchedReleasesJson = yield this.fetchReleasesFromUrl('https://sap.github.io/SapMachine/assets/data/sapmachine-releases-all.json', (0, util_1.getGitHubHttpHeaders)());
```

Comment:
```suggestion
                fetchedReleasesJson = yield this.fetchReleasesFromUrl('https://sap.github.io/SapMachine/assets/data/sapmachine-releases-all.json');
```

I think the GitHub Headers are not needed any more for the Backup URL since it is not using the GitHub API but also just downloading a plain json file from a URL.